### PR TITLE
 2 Static Pages

### PR DESCRIPTION
**Pull request by weilingooi**
ready

### CODE REVIEW

**Comment by weilingooi** [comment]
ready

File: app/assets/stylesheets/variables.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$color: #343a40;
+$hover-black: #333333
```
Comment (saobv-2223) [inline app/assets/stylesheets/variables.scss]: add endline

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <nav>
+      <ul class="nav navbar-nav navbar-right"> 
+        <li><%= link_to t("static_pages.home.homepages"), "#" %></li> 
+        <li><%= link_to t("static_pages.help.helppages"), "#" %></li> 
+        <li><%= link_to t("static_pages.about.aboutpages"), "#" %></li> 
+        <li><%= link_to t("static_pages.application.login"), "#" %></li>
+      </ul> 
+    </nav>
+  </div>
+</header>
```
Comment (saobv-2223) [inline app/views/layouts/_header.html.erb]:
```suggestion
</header>

```

File: config/settings.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+online_courses:
+  help:
+    mail: mailto:weilingooi62@gmail.com
```
Comment (saobv-2223) [inline config/settings.yml:3]: using gem figaro to manage enviroment variables

**Comment by weilingooi** [comment]
ready

**Pull request merged**